core/db/migrate/20141217215630_update_product_slug_index.rb

```diff
@@ -0,0 +1,6 @@
+class UpdateProductSlugIndex < ActiveRecord::Migration
+  def change
```

Comment:
@mbj I can get behind that on migrations that have a reasonable risk of needing to rollback the transaction, I'm not sure that this is one of those though.